src/migrations/userRolesMigration.js

```diff
@@ -0,0 +1,22 @@
+const { UserRoles } = require('../models');
+
+const listUserRoles = {
+  user: 'User',
+  admin: 'Admin',
+  superAdmin: 'Super admin'
+};
+
+const checkAvailability = async function () {
+  try {
+    Object.values(listUserRoles).forEach(async (role) => {
+      const findUserRole = await UserRoles.findOne({ name: role });
```

Comment:
Instead of multiply calls to db, try to get full list existing roles and then create only these roles, which doesn't exist. So we will have only 2 calls to db instead of 6. Also use this package for migrations https://www.npmjs.com/package/migrate-mongo (this will help us to have same pattern for whole project). So you need to install this package, create folder migrations, create file migrate-mongo-config.js... You can follow this example https://github.com/ita-social-projects/horondi_client_be/blob/master/migrate-mongo-config.js

Reply:
And everything must be covered with tests!